### DISCUSSION TITLE
Cannot embed molecules, getting unrecognized charge state error

### DISCUSSION THREAD

**Discussion by eereenah-fast** [rdkit/rdkit#5047] [Q&A]
Hi all!

I have the following two molecules: `C[S+](=O)(C)C=C(NC1=CC(=C(C=C1)Cl)Cl)[O-]` and `C[S@@+]([O-])c1ccc2[nH]cc(C=O)c(=O)c2c1`. If I try to embed any of these molecules I end up with the following error:

```
mol = Chem.MolFromSmiles('C[S+](=O)(C)C=C(NC1=CC(=C(C=C1)Cl)Cl)[O-]')
mol = Chem.AddHs(mol)
mol
```

![mol](https://user-images.githubusercontent.com/65531744/155712910-e70ec3a5-b160-4cde-9015-d9617cbf39f3.png)

```
embed_success = AllChem.EmbedMolecule(mol, useRandomCoords = True)
```
```
RDKit ERROR: [16:03:24] UFFTYPER: Unrecognized charge state for atom: 1
```

Do you know what may be causing this issue and whether there are any workarounds regarding this issue? 
Thanks in advance!

**Comment by greglandrum** [comment]
Hi @eereenah-fast : the embedding actually works despite that error message.

`EmbedMolecule()` returns the ID of the conformer added to the molecule (https://www.rdkit.org/docs/source/rdkit.Chem.rdDistGeom.html#rdkit.Chem.rdDistGeom.EmbedMolecule), so it returns -1 if it fails. For your molecule it returns 0, which is fine. 

We should improve the documentation so that this is more obvious.
